pr_check.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Run unit tests first since it activates/deactives its own virtual env
+source unit_test.sh
+
+# --------------------------------------------
+# Options that must be configured by app owner
+# --------------------------------------------
+APP_NAME="quickstarts"  # name of app-sre "application" folder this component lives in
+COMPONENT_NAME="quickstarts"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
+IMAGE="quay.io/cloudservices/quickstarts"
+
+IQE_PLUGINS="e2e"
```

Comment:
I think this is good? Generally, we use the app's name and that has to be configured in the smoke test framework that iqe uses. Not sure.